### DISCUSSION TITLE
Recognition of navigation bars

### DISCUSSION THREAD

**Discussion by Gizanik** [EDCD/EDDI#2384] [EDDI Standalone Q&A] [answered]
Good day! There is a variable {TXT:Status gui focus} which shows the current focus of the commander. Is it possible to find out on which tab the commander of a particular panel is now located?

**Comment by Tkael** [comment, marked as answer]
We can determine that you are looking at the left panel, right panel, etc. but the specific tab is not available, unfortunately.
